database/mssql/scripts/dbscripts/PSP_PIMS_S16_00/Alter Up/01_PSP_PIMS_S15_01_Alter_Up.sql

```diff
@@ -15,6 +15,27 @@ GO
 IF @@ERROR <> 0 SET NOEXEC ON
 GO
 
+declare @schema_name nvarchar(256)
+declare @table_name nvarchar(256)
+declare @col_name nvarchar(256)
+declare @Command  nvarchar(1000)
+
+set @schema_name = N'etl'
+set @table_name = N'ETL_DATA_SOURCE_TRACE'
+set @col_name = N'DATA_SOURCE_TRACE_ID'
+
+select @Command = 'ALTER TABLE ' + @schema_name + '.[' + @table_name + '] DROP CONSTRAINT ' + d.name
+ from sys.tables t
+  join sys.default_constraints d on d.parent_object_id = t.object_id
+  join sys.columns c on c.object_id = t.object_id and c.column_id = d.parent_column_id
+ where t.name = @table_name
+  and t.schema_id = schema_id(@schema_name)
+  and c.name = @col_name
+
+--print @Command
+
+execute (@Command)
```

Comment:
This is a lot of code for a relatively small issue, but basically we don't know the name of the default because it wasn't specified when it was originally created. As a result, we have to look up the name of the default programmatically and drop it.

Aqua seems to be assuming that generated Default names will be the same in every environment. This is not true.